docs/meta.md

```diff
@@ -99,6 +99,43 @@ It provides the following parameters:
 A license text that the user must accept before the snap can be
 installed.
 
+## gui/ directory
+
+The gui directory contains GUI releated files for the snap.
+
+### dekstop files
+
+The `gui/` directory may contain .desktop files for the snap. Those
+desktop files may contain all valid desktop entries from the xdg
+Desktop Entry Specification version 1.1 with some exceptions listed
+below.
```

Comment:
Would be handy to put the link in the text: https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-1.1.html